cpp/src/Ice/SSL/SchannelEngine.cpp

```diff
@@ -1204,270 +1497,6 @@ Schannel::SSLEngine::validationCallback(
             os << "SSL transport: certificate verification failure:\n" << policyStatusToString(policyStatus.dwError);
             throw SecurityException(__FILE__, __LINE__, os.str());
         }
-
-        if (!incoming && getCheckCertName() && info->peerCertificate && !host.empty())
```

Comment:
I moved this to the validation lambda callbacks created from IceSSL configuration properties.